CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * New art for Thirsting campfire option (thanks ChDaVinci)
 * New art for Withering (thanks ChDaVinci for art and Neablis for helping merge)
 * New art for Decoy (thanks EscMargot for art and Neablis for helping merge)
+* Added Demonic Coup card and placeholder art 
```

Comment:
There's a "cards" section a few lines down this should probably go in. Also, would be good if you could credit yourself here :)

Reply:
It felt weird saying "Good job me!" I figured someone else could. Will add. I wasn't sure what all needed to be updated in the changelog. Is it everything before the last release tag?